### PR TITLE
Migrate to GitHub Actions

### PR DESCRIPTION
**Pull request by petergoldstein**
This PR gets CI running with GitHub Actions.  It uses a slightly modified version of the prawnpdf/prawn config file.

It also:

1. Includes a fixture file originally imported with the prawn gem.  Paths in specs were updated
2. Addresses a couple of keyword argument ambiguities
3. Updates the CI badge in the README
4. Conditionally adds the matrix gem to the gemspec when Ruby version is 3.1.0 or higher, as matrix is no longer bundled with Ruby

Thanks to @leoarnold for his original PR and driving the CI change.

### CODE REVIEW

File: prawn-table.gemspec [hunk 1]
```diff
@@ -20,6 +20,9 @@ Gem::Specification.new do |spec|
   spec.licenses = %w(PRAWN GPL-2.0 GPL-3.0)
 
   spec.add_dependency('prawn', '>= 1.3.0', '< 3.0.0')
+
+  spec.add_dependency('matrix') if RUBY_VERSION >= '3.1.0'
```
Comment (pointlessone) [inline prawn-table.gemspec:24]: Does prawn-table use matrix gem?

Comment (petergoldstein) [reply prawn-table.gemspec:24]: This is what results from attempting to run specs on Ruby 3.1.0 when this line is omitted:

Screen Shot 2022-02-07 at 9 50 13 AM

So indirectly through the `require 'prawn'`.  If that's a development/spec only thing, we can either shift it to a development dependency or put it in the Gemfile.

Comment (pointlessone) [reply prawn-table.gemspec:24]: Oh, I see. Well, it's a transitive dependency issue. We have to bump prawn version dependency to a one that has `matrix` in its deps (a one that has not been released yet).

Comment (petergoldstein) [reply prawn-table.gemspec:24]: Ok.  So what's the follow up action?

Comment (pointlessone) [reply prawn-table.gemspec:24]: We'll have to leave Ruby 3.1 out of the matrix until after Prawn release. Either extract that bit into a separate PR and merge the rest or hold this whole PR until then. Either works for me.

**Comment by petergoldstein** [comment]
@pointlessone Removed 3.1 from CI.  This should be ready for merge.

Also, can you please update the repo so that the `continuous-integration/travis-ci` check is no longer required?  It doesn't run anymore since Travis CI.org was discontinued

**Pull request merged**